docs/data-model.md

```diff
@@ -0,0 +1,33 @@
+# TranSMART data model
+
+## TranSMART database model
+
+This diagram gives an overview of the core tables in the TranSMART database.
+The central table stores observations, which are linked to patients, concepts,
+visits, trial visits and studies.
+
+![TranSMART database diagram](model/transmart-database/transmart%20database%20diagram.jpg)
```

Comment:
The field names marked with **bold** text are not clear for me. Were they supposed to be PK/FK only or all not-nullable fields? It seems they are a bit inconsistent.

Reply:
The intention is that those in bold are the non-nullable fields.

Reply:
hmm, so it looks like some of the fields are missed, eg. observation_fact.provider_id, observation_fact.modifier_cd and e.g. concept_code.name_char is in bold, but shouldn't.

Reply:
provider_id and modifier_cd are pseudo-nullable, i.e., '@' means no provider or no modifier. I will change the name field in the concept dimension.

Reply:
It's a bit weird, they should not be part of the primary key, in my opinion. Then they can be properly nullable and have foreign key constraints.